packages/components/src/button/style.scss

```diff
@@ -210,26 +210,16 @@
 		background: none;
 		outline: none;
 		text-align: left;
-		/* Mimics the default link style in common.css */
-		color: #0073aa;
+		color: var(--wp-admin-theme-color);
 		text-decoration: underline;
 		transition-property: border, background, color;
 		transition-duration: 0.05s;
 		transition-timing-function: ease-in-out;
 		@include reduce-motion("transition");
 		height: auto;
 
-		&:hover:not(:disabled),
-		&:active:not(:disabled) {
-			color: #00a0d2;
-			box-shadow: none;
-		}
-
 		&:focus {
-			color: #124964;
-			box-shadow:
-				0 0 0 $border-width #5b9dd9,
-				0 0 var(--wp-admin-border-width-focus) $border-width rgba(30, 140, 190, 0.8);
+			border-radius: $radius-block-ui;
 		}
```

Comment:
Why did you remove all these styles? I worry about the impact

Reply:
Because those styles are from the older wp-admin styles. I don't mean to dismiss the feedback, but it's similar to when the button styles changed from bevelled to flat.